tests/targets/test_android.py

```diff
@@ -52,11 +46,57 @@ def patch_platform(platform):
     return mock.patch("buildozer.targets.android.platform", platform)
 
 
-class TestTargetAndroid:
+def init_target(temp_dir, options=None):
+    buildozer = init_buildozer(temp_dir, "android", options)
+    return TargetAndroid(buildozer)
 
-    @staticmethod
-    def default_specfile_path():
-        return os.path.join(os.path.dirname(buildozer_module.__file__), "default.spec")
+
+def call_build_package(target_android):
```

Comment:
Changed to a more "functional approach" to make it clear it only requires target_android and won't mutate self